### PR TITLE
remove confirm_email in customer create form if in admin context

### PR DESCRIPTION
**Pull request by lunika**
I don't know if it's the more sexy way to do this but I need to remove the email_confirm field in the CustomerCreateForm if we are in admin context

### CODE REVIEW

File: core/lib/Thelia/Core/EventListener/FormListener.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    /**
+     * @var ControllerResolver
+     */
+    protected $resolver;
+
+    /**
+     * @param Request $request
+     * @param ControllerResolver $resolver
+     */
+    public function __construct(Request $request, ControllerResolver $resolver)
```
Comment (lovenunu) [inline core/lib/Thelia/Core/EventListener/FormListener.php]: Why don't you use ```Symfony\Component\HttpKernel\Controller\ControllerResolverInterface``` for type checking ?

File: core/lib/Thelia/Core/EventListener/FormListener.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    {
+        $this->request = $request;
+        $this->resolver = $resolver;
+    }
+
+    public function removeEmailConfirmation(TheliaFormEvent $event)
+    {
+        $controller = $this->resolver->getController($this->request);
+
+        if ($controller[0] instanceof CustomerController) {
```
Comment (lovenunu) [inline core/lib/Thelia/Core/EventListener/FormListener.php]: Why do you check this controller ? Another one can use the form in a front context. :)

Comment (lunika) [inline core/lib/Thelia/Core/EventListener/FormListener.php]: For the front context you can set the value of ```customer_confirm_email``` to 0 or 1 if you want to verify the email or not

**Comment by lunika** [comment]
An other implementation, more easy, could be to check if the parameter ```customer_confirm_email``` is activate or not.

**Comment by lunika** [comment]
#1293 is easier and cleaner :-)

**Comment by lovenunu** [comment]
:+1:

**Pull request closed without merging**